src/modules/sim-host/sim-status.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 Intel Corporation. All rights reserved.
```

Comment:
@dlebu, should we hold it for it? I'd rather changes this to the header the rest of the files have and then we'll figure out double copyright thing.